src/validationinterface.cpp

```diff
@@ -179,19 +179,16 @@ void SyncWithValidationInterfaceQueue()
 // evaluating arguments when logging is not enabled.
 //
 // NOTE: The lambda captures all local variables by value.
-#define ENQUEUE_AND_LOG_EVENT(event, fmt, name, ...)           \
-    do {                                                       \
-        auto local_name = (name);                              \
-        LOG_EVENT("Enqueuing " fmt, local_name, __VA_ARGS__);  \
-        m_internals->m_schedulerClient.AddToProcessQueue([=] { \
-            LOG_EVENT(fmt, local_name, __VA_ARGS__);           \
-            event();                                           \
-        });                                                    \
+#define ENQUEUE_AND_LOG_EVENT(category, level, event, fmt, name, ...)                                 \
+    do {                                                                                              \
+        auto local_name = (name);                                                                     \
+        LogTrace(category, "Enqueuing " fmt "\n", local_name, __VA_ARGS__); \
```

Comment:
Indeed this should be `LogDebug()` in order to not change behavior.

The downgrade to trace should be in a different commit. Will look into this again.

>  This makes intermittent test failures harder to debug, since trace is disabled.

I'll look into what level to use. Lots of things have changed over the past few rebases.